### ISSUE TITLE
Skip first x lines before detecting header?

### ISSUE THREAD

**Issue by yooakim** [i2van/CsvLINQPadDriver#10]
I have a lot of tab separated files. So I add a folder and I then can see the files fine. The problem is that my files contain a "control record" so there is one header line followed by a datat line. Then a new header line (which is the header for the rest of the file)

Is it possible to tell CsvLINQPadDriver to skip the first _n_ lines of each file? Before it detects the header?

```
Format version  Start date      End date        Sender
5       2022-06-01      2022-06-30      Company
Country Product URI     UPC
IT      P       spotify:track:4AvvNABCDWNhrkqs0FptoN    842434515299
GB      P       spotify:track:4NuzrYl1IaAuogG9XhYOqT    084427962505
.
.
.

```

is there a way to do this without having to remove the two first lines from each line?

**Comment by i2van** [comment]
Looks like it rather tricky. I'll take a look anyway.

**Comment by i2van** [comment]
@yooakim You can use attached driver version which skip first 2 lines for each file as a temporary solution.

[2LinesSkipDriver.zip](https://github.com/i2van/CsvLINQPadDriver/files/9315801/2LinesSkipDriver.zip)

**Comment by i2van** [comment]
@yooakim Implemented in [7.3.9](https://github.com/i2van/CsvLINQPadDriver/releases/tag/7.3.9)

![image](https://user-images.githubusercontent.com/5464660/184516040-a4e5a06f-7274-4099-b95f-fab4fa85a51b.png)
